test/org/sagebionetworks/bridge/dynamodb/DynamoTaskDaoMockTest.java

```diff
@@ -33,110 +41,111 @@
+import com.google.common.collect.Maps;
 
 public class DynamoTaskDaoMockTest {
 
     private static final DateTime NOW = DateTime.parse("2015-04-12T14:20:56.123-07:00");
-    
+
     private static final String HEALTH_CODE = "AAA";
 
     private static final StudyIdentifier STUDY_IDENTIFIER = new StudyIdentifierImpl("mock-study");
-    
+
+    private static final DateTimeZone PACIFIC_TIME_ZONE = DateTimeZone.forID("America/Los_Angeles");
+
     private User user;
-    
+
     private DynamoDBMapper mapper;
-    
+
     private DynamoTaskDao taskDao;
-    
+
     @SuppressWarnings("unchecked")
     @Before
     public void before() {
         DateTimeUtils.setCurrentMillisFixed(NOW.getMillis());
-        
+
         user = new User();
         user.setHealthCode(HEALTH_CODE);
         user.setStudyKey(STUDY_IDENTIFIER.getIdentifier());
-        
+
         // This is the part that will need to be expanded per test.
         mapper = mock(DynamoDBMapper.class);
-        when(mapper.query(
-            (Class<DynamoTask>)any(Class.class), 
-            (DynamoDBQueryExpression<DynamoTask>)any(DynamoDBQueryExpression.class)
-        )).thenReturn(null);
-        
+        when(
+                        mapper.query((Class<DynamoTask>) any(Class.class),
+                                        (DynamoDBQueryExpression<DynamoTask>) any(DynamoDBQueryExpression.class)))
+                        .thenReturn(null);
+
         taskDao = new DynamoTaskDao();
         taskDao.setDdbMapper(mapper);
     }
-    
+
     @After
     public void after() {
         DateTimeUtils.setCurrentMillisSystem();
     }
-    
+
     @SuppressWarnings("unchecked")
     private void mockQuery(final List<Task> tasks) {
-        List<DynamoTask> results = Lists.newArrayList();
-        if (tasks != null) {
-            for (final Task task : tasks) {
-                results.add((DynamoTask)task);
-            }
-            when(mapper.load(any())).thenAnswer(new Answer<DynamoTask>() {
-                @Override public DynamoTask answer(InvocationOnMock invocation) throws Throwable {
-                    DynamoTask thisTask = (DynamoTask)invocation.getArguments()[0];
-                    for (Task task : tasks) {
-                        if (thisTask.getGuid().equals(task.getGuid()) && 
-                            thisTask.getHealthCode().equals(task.getHealthCode())) {
-                            return thisTask;
-                        }
+        when(mapper.load(any())).thenAnswer(new Answer<DynamoTask>() {
+            @Override
+            public DynamoTask answer(InvocationOnMock invocation) throws Throwable {
+                DynamoTask thisTask = (DynamoTask) invocation.getArguments()[0];
+                for (Task task : tasks) {
+                    if (thisTask.getGuid().equals(task.getGuid())
+                                    && thisTask.getHealthCode().equals(task.getHealthCode())) {
+                        return thisTask;
                     }
```

Comment:
Minor: In Java 8, you can use a lambda and save a bunch of boilerplate code:
            when(mapper.load(any())).thenAnswer(invocation -> {
                DynamoTask thisTask = (DynamoTask)invocation.getArguments()[0];
                for (Task task : tasks) {
                    if (thisTask.getGuid().equals(task.getGuid()) && 
                        thisTask.getHealthCode().equals(task.getHealthCode())) {
                        return thisTask;
                    }
                }
                return null;
            });